src/client/storage_estimator/SConscript

```diff
@@ -3,7 +3,7 @@
 def install_storage_estimator_module(env, lib_prefix, version):
     """Install Storage Estimador module for the specified python version"""
 
-    if GetOption("help"):
+    if GetOption('help') or GetOption('clean'):
```

Comment:
we shouldn't skip build steps when using clean.  Otherwise, clean will not work.